proxy/README.md

```diff
@@ -113,6 +113,42 @@ handled by systemd and can be viewed with:
 journalctl -u cc-proxy -f
 ```
 
+## SELinux
+
+`cc-proxy` can't run out of the box on fedora systems running SELinux enforced. 
+This is documented in issue #519
```

Comment:
Can you remove this sentence? the necessary information should be part of this README, the "how we got there" does't really matter to the end user.